app/services/support_interface/equality_and_diversity_export.rb

```diff
@@ -45,5 +46,12 @@ def application_forms
         .includes(:application_choices)
         .where.not(equality_and_diversity: nil)
     end
+
+    def provider_made_decision_on_any_application_choice?(application_form)
+      application_form.application_choices.any? do |application_choice|
+        application_choice.status.in?(%w[offer pending_conditions recruited rejected declined conditions_not_met offer_deferred]) &&
```

Comment:
This list is a bit trickier to work out than it first appears I reckon. A couple of issues:
- what if a candidate withdraws after an offer is made? (the `withdraw` status can happen with or without a decision having happened)
- does `offer_withdrawn` count as a decision (even though the provider changed there mind)?
- how will we remember to amend this list when adding/removing statuses to the workflow? Should it be a constant in `ApplicationStateChange`?

Could we work this out any other way? e.g. by the presence of a status change to `offer` or `rejected` in the audit trail?

Reply:
Hey Steve, the logic the application_choice export doesn't worry about those edge cases so i think we're okay here.

You're right though this isn't 100% ideal